sunnyportal/requests.py

```diff
@@ -239,3 +239,26 @@ def __init__(self, token, oid, date, period, interval, total=False):
 
     def handle_response(self, data):
         return responses.EnergyBalanceResponse(data)
+
+
+class LogbookRequest(RequestBase):
+    def __init__(self, token, oid, date_from, info=True, warn=True, failure=True, error=True):
```

Comment:
Consider changing warn to warning (also above in client.py).

Reply:
Will do.